src/tour/tour.js

```diff
@@ -194,6 +194,14 @@ angular.module('angular-tour.tour', [])
           scope.ttOffset = parseInt(val, 10) || tourConfig.offset;
```

Comment:
If we're axing scope.ttOffset, should we make those available to configure globally in the tourConfig service then?